docs/debian-conf.d/transport/30_vexim_ditch_spam_transport

```diff
@@ -0,0 +1,20 @@
+
+### transport/30_vexim_ditch_spam_transport
+#######################################
+
+ditch_spam_transport:
```

Comment:
You can add that prefix elsewhere as well. This way we will have naming rules and it will become very unlikely that our names would ever conflict with others.